schemas.js

```diff
@@ -108,11 +161,11 @@ const endpointSchema = (srcDir) =>
           .required()
           .max(50)
           .matches(
-            /^[a-zA-Z][a-zA-Z0-9_-]*$/,
+            /^[a-z][a-z0-9_-]*$/,
             ({ path }) =>
-              `'${path}' must start with a letter and can contain letters, numbers, underscores (_), and hyphens (-).`
+              `${path} must start with a lowercase letter and can contain lowercase letters, numbers, underscores (_), and hyphens (-).`
```

Comment:
Shall we say "can **only**" to be consistent with https://github.com/choreo-templates/choreo-source-config-validator/pull/7/files#diff-5e3395ed46163e4a024735997a51199862593814373fbb85ada54157d191a93aR146

Reply:
Done